substrate/frame/revive/rpc/Cargo.toml

```diff
@@ -22,6 +22,11 @@ name = "deploy"
 path = "examples/rust/deploy.rs"
 required-features = ["example"]
 
+[[example]]
+name = "test-deployment"
+path = "examples/rust/test-deployment.rs"
+required-features = ["example"]
```

Comment:
1. Why is this an example and not a bin? Is is a useful tool provided by this package. Not an example. Hence it should be build by CI to catch compilation errors.
2. Why guard examples behind a feature flag when they are already guarded behind `--example`?
3. Same questions for `deploy` above.

Reply:
1 right could be a bin, I thought that both would be checked by CI anyway, in my mind they are mostly similar, but I used example as it's just meant to be used for some testing unlike 'eth-rpc' and 'eth-indexer' 
2 required-features specify which feature the target needs, here I need the example that has utilities to send transactions

Reply:
Please move both to bin and remove feature flag if you don't mind. I would like all code to be built.

Reply:
thanks i hate features